autoconfigure/src/main/java/mesosphere/marathon/client/Marathon.java

```diff
@@ -0,0 +1,108 @@
+package mesosphere.marathon.client;
+
+import java.util.List;
+import java.util.Map;
+
+import feign.QueryMap;
+import mesosphere.marathon.client.model.v2.App;
+import mesosphere.marathon.client.model.v2.DeleteAppTaskResponse;
+import mesosphere.marathon.client.model.v2.DeleteAppTasksResponse;
+import mesosphere.marathon.client.model.v2.Deployment;
+import mesosphere.marathon.client.model.v2.GetAppResponse;
+import mesosphere.marathon.client.model.v2.GetAppTasksResponse;
+import mesosphere.marathon.client.model.v2.GetAppsResponse;
+import mesosphere.marathon.client.model.v2.GetEventSubscriptionRegisterResponse;
+import mesosphere.marathon.client.model.v2.GetEventSubscriptionsResponse;
+import mesosphere.marathon.client.model.v2.GetServerInfoResponse;
+import mesosphere.marathon.client.model.v2.GetTasksResponse;
+import mesosphere.marathon.client.model.v2.Group;
+import mesosphere.marathon.client.model.v2.QueueResponse;
+import mesosphere.marathon.client.model.v2.Result;
+import mesosphere.marathon.client.MarathonException;
+
+import feign.Param;
+import feign.RequestLine;
+
+public interface Marathon {
+    // Apps
+    @RequestLine("GET /v2/apps")
+    GetAppsResponse getApps() throws MarathonException;
+
+    @RequestLine("GET /v2/apps")
+    GetAppsResponse getApps(@QueryMap Map<String, String> queryMap) throws MarathonException;
```

Comment:
Added a new method to support passing query parameters to /v2/apps
e.g.
/v2/apps?id=myServiceName

Submitted a PR to mesosphere/marathon-client for this change:
https://github.com/mesosphere/marathon-client/pull/44